docs/releases/1.10.1.txt

```diff
@@ -61,3 +61,6 @@ Bugfixes
 
 * Fixed ``contrib.admindocs`` crash when a view is in a class, such as some of
   the admin views (:ticket:`27018`).
+
+* Fixed the creation of ``ContentType`` and ``Permission`` objects when calling
```

Comment:
Actually I can reproduce without using ``--run-syncdb`` but it only affects apps without migrations.